pom.xml

```diff
@@ -70,11 +70,7 @@
             <artifactId>jakarta.persistence-api</artifactId>
```

Comment:
always fetch and pull from `main` and then branch from `main` before making any changes.